### PR TITLE
GOV.UK Framework | Staff Record | Days Sickness

### PR DESCRIPTION
**Pull request by ChazUK**

### CODE REVIEW

**Comment by ChazUK** [comment]
[Staff Record - Days Sickness](https://trello.com/c/ak4lYFkI/533-staff-record-days-sickness)

File: src/app/features/workers/days-of-sickness/days-of-sickness.component.html [hunk 1]
```diff
@@ -1,51 +1,77 @@
...
+            </ng-container>
+          </span>
+          <input
+            class="govuk-input govuk-date-input__input govuk-input--width-4"
+            [class.govuk-input--error]="submitted && form.get('days').invalid"
+            formControlName="days"
+            id="days"
+            name="days"
+            type="number"
+            pattern="^([0-9]*[.])?[0-9]+$"
```
Comment (omarCreativeDev) [inline src/app/features/workers/days-of-sickness/days-of-sickness.component.html]: is it possible to move this regex to the constants file?

File: src/app/features/workers/days-of-sickness/days-of-sickness.component.ts [hunk 2]
```diff
@@ -1,133 +1,106 @@
...
 import { Contracts } from '@core/constants/contracts.enum';
-import { Worker } from '@core/model/worker.model';
-import { MessageService } from '@core/services/message.service';
-import { WorkerEditResponse, WorkerService } from '@core/services/worker.service';
-import { Subscription } from 'rxjs';
-import { take } from 'rxjs/operators';
+import { BackService } from '@core/services/back.service';
+import { ErrorSummaryService } from '@core/services/error-summary.service';
+import { WorkerService } from '@core/services/worker.service';
 import { isNull } from 'util';
```
Comment (omarCreativeDev) [inline src/app/features/workers/days-of-sickness/days-of-sickness.component.ts]: is this being used anywhere?

Comment (ChazUK) [reply src/app/features/workers/days-of-sickness/days-of-sickness.component.ts]: Updated to use lodash isNull instead

File: src/app/features/workers/days-of-sickness/days-of-sickness.component.html (hunk 1 above)
Comment (ChazUK) [reply src/app/features/workers/days-of-sickness/days-of-sickness.component.html]: Moved to constants

**Pull request merged**